src/bg/on-user-script-xhr.js

```diff
@@ -118,18 +120,46 @@ function open(xhr, d, port) {
       }
     }
   }
+  
+  // if same-origin XHR, add cookies unless already specified by user.
```

Comment:
All comments should be grammatically correct sentences, including capitalization.